test/versioning/maven.spec.js

```diff
@@ -70,3 +77,56 @@ describe('versioning/maven/compare', () => {
     expect(compare('1', '1-cr1')).toEqual(1);
   });
 });
+
+describe('versioning/maven/index', () => {
+  it('validates version string', () => {
+    expect(isVersion('')).toEqual(false);
+    expect(isVersion('v1.0.0')).toEqual(false);
```

Comment:
In npm and many other package managers, this would return `true`. Let's try to double check for Maven somehow.

Reply:
Okay, i'll fix it today

Reply:
I tried searching and haven't found any examples of versions like 'v1.0.0' in the docs, so maybe it's not allowed.